src/FileFormats/NL/gen.jl

```diff
@@ -0,0 +1,16 @@
+# This script builds the list of recognized ASL opcodes using the header files
+# in ASL_jll. Only re-run it if AMPL adds new opcodes (which is unlikely).
```

Comment:
This file is not included right ?  Maybe it could be outside `src` ? Or make it clear that it's not included in this header ? I was a bit confused.

Reply:
Yeah it's just a helper script to rebuild the OPCODEs.